src/model/wallet.ts

```diff
@@ -134,18 +252,21 @@ export const signTransaction = async ({ transaction }) => {
 };
 
 export const signMessage = async (
-  message,
+  message: Arrayish | Hexable,
   authenticationPrompt = lang.t('wallet.authenticate.please')
-) => {
+): Promise<null | string> => {
   try {
     logger.sentry('about to sign message', message);
     const wallet = await loadWallet(authenticationPrompt);
     try {
-      const signingKey = new ethers.utils.SigningKey(wallet.privateKey);
-      const sigParams = await signingKey.signDigest(
-        ethers.utils.arrayify(message)
-      );
-      return ethers.utils.joinSignature(sigParams);
+      if (wallet) {
+        const signingKey = new ethers.utils.SigningKey(wallet.privateKey);
+        const sigParams = await signingKey.signDigest(
+          ethers.utils.arrayify(message)
```

Comment:
could this be technically Arrayish | Hexable | BigNumberish (based on arrayify doc here: https://docs.ethers.io/v4/api-utils.html)

Reply:
My linter complains if I pass BigNumberish, but I was missing the String type

![Screen Shot 2020-08-13 at 4 13 23 PM](https://user-images.githubusercontent.com/1247834/90182457-00f79780-dd80-11ea-98fe-215c6d23a1ec.png)